lexicons/app/bsky/graph/listitem.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.listitem",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "An item under a declared list of actors",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["subject", "list", "createdAt"],
+        "properties": {
+          "subject": {"type": "string", "format": "did"},
+          "list": {"type": "string", "format": "at-uri"},
```

Comment:
Can we get an optional `reason` field here which is an object of a text string (max 1k chars) and a richtext facets? We want users to be able to explain why somebody is on list

Reply:
yeah sure 👍